public/app/features/admin/Users/AnonUsersTable.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useMemo } from 'react';
+
+import {
+  Avatar,
+  CellProps,
+  Column,
+  FetchDataFunc,
+  InteractiveTable,
+  Pagination,
+  Stack,
+  Badge,
+  Tooltip,
+} from '@grafana/ui';
+import { UserAnonymousDeviceDTO } from 'app/types';
+
+type Cell<T extends keyof UserAnonymousDeviceDTO = keyof UserAnonymousDeviceDTO> = CellProps<
+  UserAnonymousDeviceDTO,
+  UserAnonymousDeviceDTO[T]
+>;
+
+// A helper function to parse the user agent string and extract parts
+const parseUserAgent = (userAgent: string) => {
+  const parts = {
+    browser: userAgent.split(' ')[0],
+    computer: userAgent.split(' ')[1],
+  };
+  return parts;
+};
+
+// A helper function to truncate each part of the user agent
+const truncatePart = (part: string, maxLength: number) => {
```

Comment:
We have [this utility](https://github.com/grafana/grafana/blob/main/public/app/features/plugins/extensions/utils.tsx#L245) that could be used here.